Jenkinsfile

```diff
@@ -78,7 +78,14 @@ spec:
                 input 'ok to test?'
             }
         }
-        stage('Build/Test/Lint') {
+        stage('Build/Lint') {
+            steps {
+                // The build task includes check, and assemble.  Linting happens during the check
+                // task and uses the spotless gradle plugin.
+                sh "./gradlew --no-daemon build -x test"
+            }
+        }
+        stage('Test/Lint') {
```

Comment:
An alternative is to use the flag ` --no-parallel` to avoid running the tests in parallel. Wdyt?